lib/cadet_web/admin_views/admin_grading_view.ex

```diff
@@ -38,6 +40,14 @@ defmodule CadetWeb.AdminGradingView do
         for submission <- submissions do
           user = users |> Enum.find(&(&1.id == submission.student_id))
           assessment = assessments |> Enum.find(&(&1.id == submission.assessment_id))
+          team = teams |> Enum.find(&(&1.id == submission.team_id))
+          team_members = team_members |> Enum.filter(&(&1.team_id == submission.team_id))
+
+          team_member_users =
+            team_members
+            |> Enum.map(fn team_member ->
+              users |> Enum.find(&(&1.id == team_member.student_id))
```

Comment:
I just realised this results in an `n+1` query problem. We should use a join instead to keep it to O(1) total queries.

Reply:
Actually, on second thought, should be fine as there are no team members for now, but we might want to look at it again in the future. Will file an issue.